test/services/user/registration.test.js

```diff
@@ -4,32 +4,45 @@ const assert = require('assert');
 const app = require('../../../src/app');
 const accountModel = require('../../../src/services/account/model');
 const { consentModel } = require('../../../src/services/consent/model');
-const { userModel, registrationPinModel } = require('../../../src/services/user/model');
+const {
+	userModel,
+	registrationPinModel,
+} = require('../../../src/services/user/model');
 const { schoolModel } = require('../../../src/services/school/model');
 
 const registrationService = app.service('registration');
 const registrationPinService = app.service('registrationPins');
 const testObjects = require('../helpers/testObjects')(app);
 
-const patchSchool = (system, schoolId) => schoolModel.findOneAndUpdate({ _id: schoolId }, {
-	$push: {
-		systems: system._id,
-	},
-}, { new: true }).lean().exec();
+const patchSchool = (system, schoolId) =>
+	schoolModel
+		.findOneAndUpdate(
+			{ _id: schoolId },
+			{
+				$push: {
+					systems: system._id,
+				},
+			},
+			{ new: true }
+		)
+		.lean()
+		.exec();
 
-const createAccount = (system) => accountModel.create({
-	lasttriedFailedLogin: '1970-01-01T00:00:00.000+0000',
-	activated: false,
-	username: 'fritz',
-	password: '',
-	systemId: system._id,
-});
+const createAccount = (system) =>
```

Comment:
Codacy found an issue: ['createAccount' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)